mpas_analysis/ocean/meridional_overturning_circulation.py

```diff
@@ -388,8 +381,15 @@ def _compute_moc_postprocess(config, streams, variableMap, calendar,
     latAtlantic = mocDictClimo['latAtlantic']['data']
     dLat = latAtlantic - 26.5
     indlat26 = np.where(dLat == np.amin(np.abs(dLat)))
-    nyears = (dictTseries['endYearTseries'] -
-              dictTseries['startYearTseries'] + 1)
+
+    startYear = days_to_datetime(ds.Time.min()).year
+    endDate = days_to_datetime(ds.Time.max())
+    endYear = endDate.year
+    if endDate.month < 12:
+        endYear -= 1
+    nyears = (endYear - startYear + 1)
+
+    Time = np.zeros(nyears*12)
```

Comment:
Ah, I see now that I think the problem is more serious than that.  You're trying to have 2 different variables `Time` that do different things.